src/index.js

```diff
@@ -0,0 +1,66 @@
+import 'lodash';
+import './style.css';
+import sync from './assets/icons/sync.png';
+import downLeft from './assets/icons/down_left.png';
+import menuIcon from './assets/icons/menuIcon.png';
+
+const todoContaiter = document.querySelector('.todoContaiter');
+const titleContainer = document.querySelector('.title-list');
+const descriptionContainer = document.querySelector('.description-list');
+
+const tasksArray = [
+  {
+    index: '',
+    description: 'first task',
+    completed: false,
+  },
+  {
+    index: '',
+    description: 'second task',
+    completed: false,
+  },
+  {
+    index: '',
+    description: 'third task',
+    completed: false,
+  },
+];
+
+tasksArray.forEach((task) => {
+  const taskHtml = `<ul class="task-list">
+  <li class="task-list-item">
+    <input type="checkbox" name="check-task" id="check"/>
+  </li>
+  <li class="task-list-item">${task.description}</li>
+  </li>
+  <li class="task-list-item">
+    <img class="icon" src=${menuIcon} />
+  </li>
+</ul>`;
+  todoContaiter.innerHTML += taskHtml;
+});
+
+const titleHtml = `
+  <li class="title-list-item">
+    <span>Today's To Do</span>
+  </li>
+  <li class="title-list-item">
+    <img class="icon" src=${sync} alt="">
+  </li>
+`;
+titleContainer.innerHTML = titleHtml;
+
+const descriptionHtml = `
+  <li class="description-list-item first-child">
+    <input
+    type="text"
+    name="task-description"
+    id="description"
+    placeholder="Add to your list..."
+    />
+  </li>
+  <li class="description-list-item second-child">
+    <img class="icon" src=${downLeft} />
+  </li>
+`;
+descriptionContainer.innerHTML = descriptionHtml;
```

Comment:
- [ ] Kindly wrap the code which populates the HTML in a function, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Write%20a%20function%20to%20iterate%20over%20the%20tasks%20array%20and%20populate%20an%20HTML%20list%20item%20element%20for%20each%20task.) requirement.

- [ ] Please make sure that the list of tasks are sorted according to their individual index values before populating the HTML, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.) requirement.

Reply:
Correction done